app/scripts/modules/core/pipeline/config/stages/runJob/runJobStage.js

```diff
@@ -16,7 +16,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.runJobStage', [
       label: 'Run Job',
       description: 'Runs the previously baked or found image',
       strategyDescription: 'Runs the image specified',
-      key: 'runJob',
+      key: 'run',
```

Comment:
That is a very, um, terse key for a stage type. But if you're good with it, I'm good with it.